pkg/scheduler/api/compatibility/compatibility_test.go

```diff
@@ -1252,13 +1257,15 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		wantPredicates.Insert("CheckNodeCondition") // mandatory predicate
 		for _, p := range tc.ExpectedPolicy.Predicates {
 			wantPredicates.Insert(p.Name)
+			seenPredicates.Insert(p.Name)
```

Comment:
We have this information in `schedPredicates` already.

Reply:
`schedPredicates` are cleared after each test, `seenPredicates` aren't

Reply:
Uhm.... that sounds wrong. Can you think of a reason why is that?

Reply:
Uhm... this is weird. It looks like it is testing the test. i.e. making sure that the tests cover all the cases.
In any case, I think its more clear if `seenPredicates` is obtained `seenPredicates.Union(schedPredicates)` in each test.

Reply:
sg, done!